client/blocks/reader-recommended-sites/index.jsx

```diff
@@ -41,11 +41,8 @@ export class RecommendedSites extends React.PureComponent {
 	};
 
 	render() {
-		const { sites, followSource } = this.props;
-
-		if ( isEmpty( sites ) ) {
-			return null;
-		}
+		const { followSource } = this.props;
+		const sites = ! isEmpty( this.props.sites ) ? this.props.sites : [ {}, {} ];
```

Comment:
Worth adding a comment here to explain that `[ {}, {} ]` is to support placeholders?

Reply:
hmm. really good point, its super not-obvious.
I'll make an explicit variable instead:
```js
const placholderSites = [ {}, {} ];
...
```